test/showcase-echo-client/protos/google/showcase/v1beta1/sequence.proto

```diff
@@ -84,7 +84,7 @@ service SequenceService {
     post: "/v1beta1/{name=streamingSequences/*}:stream"
       body: "*"
     };
-    option (google.api.method_signature) = "name,content,error";
```

Comment:
Quick question, don't we still need content so we know what was being sent ?

Reply:
Oh dang, good catch, I'm not sure why that's not there in the regen. Let me look at the gapic showcase PR

Reply:
It looks like the adding of the fail index didn't change this https://github.com/googleapis/gapic-showcase/pull/1364/files

it was instead done by other maintainers because they were considered "non-exist" fields in the method signature https://github.com/googleapis/gapic-showcase/pull/1315 

It hasn't changed any functionality for the tests, things still function normally!